render_app.conf.dist

```diff
@@ -6,6 +6,7 @@
   webworkJWTsecret => 'private',
   SITE_HOST => 'http://localhost:3000',
   CORS_ORIGIN => '*',
+  STATIC_EXPIRES => 86400,
```

Comment:
I missed when this was added, or I would have commented before this was merged.

This absolutely should not be the default setting.  The default setting here should be undefined.  

@taniwallach:  You continue to ignore the facts that I have been telling you about what cache-control is.  It does not at all do what you thing it does.  Generally speaking, websites should never use it.  Browser caching is there for a reason.  It reduces how much the client needs to download from the server.  Furthermore, your claims in your statements about this pull request and what it does are absolutely and entirely false.  This does not in any way reduce server load.  In fact, it increases server load.  Setting the cache-control to expire 24 hours in the future is also a big no-no for a website.  This means that your users will be forced to download the static content every single day.  That content will rarely change.  Typically something like once a year.  Listen for once.  I have told you all of this several times already.